### PR TITLE
[ADD] 6.1 repo meta files

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="6.1" ODOO_REPO="odoo/odoo"
+  - VERSION="6.1" ODOO_REPO="OCA/OCB"
```
Comment (pedrobaeza) [inline .travis.yml]: There's no OCB version 6.1 on GitHub, so please remove it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[![Build Status](https://travis-ci.org/OCA/hr-timesheet.svg?branch=6.1)](https://travis-ci.org/OCA/hr-timesheet)
+[![Coverage Status](https://img.shields.io/coveralls/OCA/hr-timesheet.svg)](https://coveralls.io/r/OCA/hr-timesheet?branch=6.1)
```
Comment (pedrobaeza) [inline README.md]: Please use PNG version for coveralls to get the exact flag for the version:

```
[![Coverage Status](https://coveralls.io/repos/OCA/hr-timesheet/badge.png?branch=6.1)](https://coveralls.io/r/OCA/hr-timesheet?branch=6.1)
```

Comment (eLBati) [inline README.md]: Changes done.
Thanks

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**